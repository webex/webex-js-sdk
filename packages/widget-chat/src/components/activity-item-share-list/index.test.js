import React from 'react';
import renderer from 'react-test-renderer';

import ActivityItemPost from '.';

describe(`ActivityItemPost post component`, () => {
  const activity = {
    id: `test-123-123-123-123`,
    isSelf: true,
    content: `Test Activity Content`,
    name: `Test User`,
    timestamp: `2016-09-20T19:52:57.186Z`,
    verb: `post`
  };

  const component = renderer.create(
    <ActivityItemPost {...activity} />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
