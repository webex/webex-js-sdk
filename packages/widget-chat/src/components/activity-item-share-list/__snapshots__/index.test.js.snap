exports[`ActivityShareList post component renders properly 1`] = `
<div
  className="activity-item">
  <div
    className="avatar-wrapper">
    <div
      className="avatar"
      style={undefined}>
      <span
        className="avatar-letter">
        T
      </span>
    </div>
  </div>
  <div
    className="content-container">
    <div
      className="meta">
      <div
        className="display-name"
        title="{name}">
        Test User
      </div>
      <div
        className="published">
        2016-09-20T19:52:57.186Z
      </div>
    </div>
    <div
      className="activity-content">
      <div
        className="activity-share-list">
        <div
          className="activity-share-item">
          <div
            className="share-file-icon">
            <div
              className="icon"
              title={undefined} />
          </div>
          <div
            className="share-meta">
            <div
              className="share-file-info">
              <div
                className="share-name">
                testImage.js
              </div>
              <div
                className="share-file-props">
                <span
                  className="share-file-size">
                  0 Bytes
                </span>
                <span
                  className="share-file-type" />
              </div>
            </div>
            <div
              className="share-item-actions">
              <div
                className="share-action-item">
                <button
                  className="icon-button"
                  onClick={[Function]}
                  onKeyUp={[Function]}>
                  <div
                    className="icon"
                    title="Download this file" />
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="activity-post-actions">
    <div
      className="activity-post-action">
      <div
        className="post-action-item">
        <button
          className="icon-button"
          onClick={[Function]}
          onKeyUp={[Function]}>
          <div
            className="icon"
            title="Flag this message" />
        </button>
      </div>
    </div>
    <div
      className="activity-post-action">
      <div
        className="action-spacer" />
    </div>
  </div>
</div>
`;
